interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js

```diff
@@ -241,514 +309,718 @@ var fnRef;
+        }
+        ie5=true;
+    }
+    if (ns6) {
+        ns6=false;
+    }
+}
+if ((ns4)||(ie4)||(ns6)) {
+    document.onmousemove=mouseMove
+    if (ns4) {
+        document.captureEvents(Event.MOUSEMOVE)
+    } } else {
+    overlib=no_overlib;
+    nd=no_overlib;
+    ver3fix=true;
+    }
+    function no_overlib()
+    {
+        return ver3fix;
+    }
+    function overlib()
+    {
+        otext=ol_text;
+        ocap=ol_cap;
+        osticky=ol_sticky;
+        obackground=ol_background;
+        oclose=ol_close;
+        ohpos=ol_hpos;
+        ooffsetx=ol_offsetx;
+        ooffsety=ol_offsety;
+        ofgcolor=ol_fgcolor;
+        obgcolor=ol_bgcolor;
+        otextcolor=ol_textcolor;
+        ocapcolor=ol_capcolor;
+        oclosecolor=ol_closecolor;
+        owidth=ol_width;
+        oborder=ol_border;
+        ostatus=ol_status;
+        oautostatus=ol_autostatus;
+        oheight=ol_height;
+        osnapx=ol_snapx;
+        osnapy=ol_snapy;
+        ofixx=ol_fixx;
+        ofixy=ol_fixy;
+        ofgbackground=ol_fgbackground;
+        obgbackground=ol_bgbackground;
+        opadxl=ol_padxl;
+        opadxr=ol_padxr;
+        opadyt=ol_padyt;
+        opadyb=ol_padyb;
+        ofullhtml=ol_fullhtml;
+        ovpos=ol_vpos;
+        oaboveheight=ol_aboveheight;
+        ocapicon=ol_capicon;
+        otextfont=ol_textfont;
+        ocaptionfont=ol_captionfont;
+        oclosefont=ol_closefont;
+        otextsize=ol_textsize;
+        ocaptionsize=ol_captionsize;
+        oclosesize=ol_closesize;
+        otimeout=ol_timeout;
+        ofunction=ol_function;
+        odelay=ol_delay;
+        ohauto=ol_hauto;
+        ovauto=ol_vauto;
+        ocloseclick=ol_closeclick;
+        ocss=ol_css;
+        ofgclass=ol_fgclass;
+        obgclass=ol_bgclass;
+        otextfontclass=ol_textfontclass;
+        ocaptionfontclass=ol_captionfontclass;
+        oclosefontclass=ol_closefontclass;
+        opadunit=ol_padunit;
+        oheightunit=ol_heightunit;
+        owidthunit=ol_widthunit;
+        otextsizeunit=ol_textsizeunit;
+        otextdecoration=ol_textdecoration;
+        otextstyle=ol_textstyle;
+        otextweight=ol_textweight;
+        ocaptionsizeunit=ol_captionsizeunit;
+        ocaptiondecoration=ol_captiondecoration;
+        ocaptionstyle=ol_captionstyle;
+        ocaptionweight=ol_captionweight;
+        oclosesizeunit=ol_closesizeunit;
+        oclosedecoration=ol_closedecoration;
+        oclosestyle=ol_closestyle;
+        ocloseweight=ol_closeweight;
+        fnRef='';
+        if ((ns4)||(ie4)||(ns6)) {
+            if (over) {
+                cClick();
+            }
+            oframe=ol_frame;
+            if (ns4) {
+                over=oframe.document.overDiv
+                if (ie4) {
+                    over=oframe.overDiv.style
+                    if (ns6) {
+                        over=oframe.document.getElementById("overDiv");
+                    }
+                }
+            }
```

Comment:
oh, wait, I think I found broken code. I am pretty sure this shouldn't be nested.

Reply:
okay, think i forgot to use the ci.sh script to select files locally and instead fed it whole directories; will explore tomorrow

Reply:
trying 
`$HOME/.composer/vendor/bin/phpcs -p -n --extensions=php,inc --report-diff=changes.diff --standard=ci/phpcs.xml .`
then can 
`patch -p0 -ui changes.diff`

Reply:
patch command throws an error will try instead
`$HOME/.composer/vendor/bin/phpcbf -p -n --extensions=php,inc --standard=ci/phpcs.xml .`